src/pattern-library/util/jsx-to-string.tsx

```diff
@@ -3,18 +3,26 @@ import hljsXMLLang from 'highlight.js/lib/languages/xml';
 import hljsJavascriptLang from 'highlight.js/lib/languages/javascript';
 import { Fragment } from 'preact';
 
+import type { ComponentChildren, VNode } from 'preact';
+
 /**
  * Escape `str` for use in a "-quoted string.
- *
- * @param {string} str
  */
-function escapeQuotes(str) {
+function escapeQuotes(str: string) {
   return str.replace(/"/g, '\\"');
 }
 
-/** @param {any} type */
-function componentName(type) {
-  const name = typeof type === 'string' ? type : type.displayName ?? type.name;
+/**
+ * Format a component's name for display
+ */
+function componentName(type: any) {
+  let name = typeof type === 'string' ? type : type.displayName ?? type.name;
+
+  // Remove the string "Wrapper" from component names. This allows the pattern
+  // library to create a convenience wrapper component around a given component
+  // being documented, and have it appear that the documented component is being
+  // used directly in rendered source content.
+  name = name.replace('Wrapper', '');
```

Comment:
```suggestion
  name = name.replace(/\BWrapper$/, '');
```

Matches "Wrapper" only at the end of the component name, and not if the component name is exactly "Wrapper".